spring/src/main/java/org/axonframework/spring/config/SpringAxonAutoConfigurer.java

```diff
@@ -134,7 +137,9 @@ private void registerSagaBeanDefinitions(Configurer configurer, BeanDefinitionRe
         for (String saga : sagas) {
             Saga sagaAnnotation = beanFactory.findAnnotationOnBean(saga, Saga.class);
             SagaConfiguration<?> sagaConfiguration = SagaConfiguration.subscribingSagaManager(beanFactory.getType(saga));
-            // TODO: Register resource injector here.
+
+            sagaConfiguration.configureResourceInjector(c -> beanFactory.getBean(ResourceInjector.class));
```

Comment:
This isn't necessary anymore if the sagaConfiguration uses the global config injector by default